src/Core/DevOps/Locust/setup.php

```diff
@@ -86,7 +86,8 @@
 WHERE
   route_name = 'frontend.detail.page' AND
   is_deleted = 0 AND
-  is_canonical = 1
+  is_canonical = 1 AND
+  language_id = unhex('2fbb5fe2e29a4d70aa5854ce7ce3e20b')
```

Comment:
This should be default language of the sales_channel. Change the selects above to get that information here

Also this is needed on the listing pages too? 🤔